packages/syft/src/syft/client/client.py

```diff
@@ -656,6 +659,9 @@ def login(
     if email and password:
         login_credentials = UserLoginCredentials(email=email, password=password)
 
+    if login_credentials is None:
```

Comment:
I think logging in without credentials is a bit confusing. I think being explicit about creating a guest account isnt a bad thing. If we want to keep this we should at least inform the user about what is happening

Reply:
+1 : MAYBE: a print statement: 
Logging into {DOMAIN NAME} ... as GUEST

Reply:
what would even happen when we would login as a guest while guest registration is disabled?

Reply:
@koenvanderveen as a guest user (without registration) we can still explore datasets on the domain.